integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkTableInfo.java

```diff
@@ -98,7 +102,7 @@ static SparkTableInfo create(SparkBaseTable baseTable) {
     sparkTableInfo.tableName = items[1];
     sparkTableInfo.database = items[0];
     sparkTableInfo.columns =
-        Arrays.stream(baseTable.schema().fields())
+        Arrays.stream(getSchema(baseTable).fields())
```

Comment:
why change this line? is `baseTable.schema()` will return the schema from spark connector not Gravitino ?

Reply:
using `baseTable.schema()` directly will failed as the `schema` method is `Deprecated`
![image](https://github.com/datastrato/gravitino/assets/94670132/d38038d9-08e3-487b-9d6b-ed82fd55a313)

Reply:
> ok, please add come comment

ok

Reply:
done